components/formstack/actions/create-form-submission/create-form-submission.mjs

```diff
@@ -0,0 +1,51 @@
+import formstack from "../../formstack.app.mjs";
+
+export default {
+  type: "action",
+  key: "formstack-create-form-submission",
+  name: "Create Form Submission",
+  description: "Create a new submission for the specified form. [See docs here](https://formstack.readme.io/docs/form-id-submission-post)",
+  version: "0.0.1",
+  props: {
+    formstack,
+    formId: {
+      propDefinition: [
+        formstack,
+        "formId",
+      ],
+    },
+    fields: {
+      label: "Fields",
+      description: "Value that should be stored for a specific field on the form. For this parameter, x must contain the id of the field whose value should be set. For fields with subfields (name, address, etc), use array notation with each item being the subfield and each key being the name of that subfield. E.g. { \"field_12345\": { \"first\": \"John\", \"last\": \"Smith\" } }",
+      type: "string",
+    },
+    encryptionPassword: {
+      label: "Encryption Password",
+      description: "The password used to decrypt your submissions. Without this value, you will only receive the submission ID and the success message when submitting encrypted forms",
+      type: "string",
+      optional: true,
+    },
+  },
+  async run({ $ }) {
+    const {
+      formId,
+      fields,
+      encryptionPassword,
+    } = this;
+
+    const parsedFields = JSON.parse(fields);
```

Comment:
If user just input and empty string or a random character here seems like he is getting this error
```
Uncaught SyntaxError: Unexpected end of JSON input
```

Reply:
Have a default way (or an example) to validate this ?

Reply:
The way I usually test this is by using the UI. Just by input some space in the input box.